src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -194,7 +194,7 @@
     public final OpOrder readOrdering = new OpOrder();
 
     /* This is used to generate the next index for a SSTable */
-    private final AtomicInteger fileIndexGenerator = new AtomicInteger(0);
+    private final java.util.function.Supplier<? extends SSTableUniqueIdentifier> fileIndexGenerator;
```

Comment:
Nit: maybe we can extend the import of `com.google.common.base.*` to:
```
import com.google.common.base.Function;
import com.google.common.base.Joiner;
import com.google.common.base.Predicate;
import com.google.common.base.Predicates;
```
And import `java.util.function.Supplier` to use just `Supplier` without the qualified class name.

Reply:
oh, interesting, it must a leftover, thanks